x-pack/plugins/security_solution/public/detections/configurations/security_solution_detections/render_cell_value.tsx

```diff
@@ -116,6 +118,17 @@ export const RenderCellValue: React.FC<NonNullable<EuiDataGridCellProps['cellCon
         ?.value?.[0] as number | undefined;
       return ecsSuppressionCount ? parseInt(ecsSuppressionCount, 10) : dataSuppressionCount;
     }, [ecsData, data]);
+    const panels = useExpandableFlyoutState();
```

Comment:
By using this state here, any time some part of state in the flyout changes, every cell in every table will re-render, and that will be pretty impactful on performance. Should move this to here https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx#L259 or in the GuidedTour component itself to avoid this

Reply:
This branch:
![tour_rerenders](https://github.com/elastic/kibana/assets/56408403/0e20db31-46c8-4b73-9694-9166ab54af7c)
Main:
![tour_rerenders_main](https://github.com/elastic/kibana/assets/56408403/dc4946bb-d741-4130-8a33-953abf9a4519)

Reply:
There was some code didn't get cleaned up properly , fixed in https://github.com/elastic/kibana/pull/181442/commits/6ef8cae98c743c8e307d3ea658e08ef73045fa1a